knotx-core/src/main/java/io/knotx/fragments/SnippetPatterns.java

```diff
@@ -38,13 +45,24 @@ public SnippetPatterns(SnippetOptions snippetOptions) {
         .compile(String
             .format(SNIPPET_PATTERN, snippetOptions.getTagName(), snippetOptions.getParamsPrefix(),
                 snippetOptions.getTagName()), Pattern.DOTALL);
+    snippetWithFallbackPattern = Pattern
+        .compile(String
+            .format(SNIPPET_WITH_FALLBACK_PATTERN, snippetOptions.getTagName(), snippetOptions.getParamsPrefix(),
+                snippetOptions.getParamsPrefix(), snippetOptions.getTagName()), Pattern.DOTALL);
+    fallbackPattern = Pattern
+        .compile(String
+            .format(FALLBACK_PATTERN, snippetOptions.getFallbackTagName(),snippetOptions.getFallbackTagName()), Pattern.DOTALL);
   }
 
-  public Pattern getAnySnippetPattern() {
-    return anySnippetPattern;
-  }
+  public Pattern getAnySnippetPattern() { return anySnippetPattern; }
```

Comment:
Please make it 3-liner again (as the methods below).
Now it is inconsistent with google formatting rules for java that we follow (see more in [contributing guide](https://github.com/Cognifide/knotx/blob/master/CONTRIBUTING.md#coding-conventions))

Reply:
filer reformatted